### DISCUSSION TITLE
Is there a way to configure detekt-formatting plugin for tabs indentation?

### DISCUSSION THREAD

**Discussion by dunkpi** [detekt/detekt#6762] [Q&A] [answered]
I want to use tabs indentation and I wasn't able to find a clear way to set indentation style in config.

Is there any?

**Comment by schalkms** [comment, marked as answer]
No, detekt and the detekt-formatting wrapper does not support all ktlint features.
See #4962 for more details.

Anyways, you could write a [custom detekt rule](https://detekt.dev/docs/introduction/extensions), which achieves tabs as indentation style.
